### ISSUE TITLE
Testing markdown link rename: moving file with links and linked file doesn't update linked file

### ISSUE THREAD

**Issue by andreamah** [microsoft/vscode#164644]
Testing #164522

1. Have the file with links and linked files in different directories.
2. Move them both to the same directory in the same action.
3. The target link doesn't refresh :bug:

![Recording 2022-10-25 at 12 11 01](https://user-images.githubusercontent.com/31675041/197861444-c7c945a6-5ff6-46cc-957a-14d01c4c42cd.gif)

**Comment by andreamah** [comment]
I'm still seeing this happening (at least on Linux)?

**Comment by mjbvz** [comment]
@andreamah `.js` files don't participate in markdown link updates automatically. Can you test this by either:

- Moving the markdown file and an image file, such as a `.png`
- Add `.js` to the `markdown.updateLinksOnFileMove.include` setting. You can add an entry for `**/*.js` for this

Both of these fix the issue for me

---

I agree this case is confusing, but it would also be confusing IMO if we ignored `markdown.updateLinksOnFileMove.include` in this case. If lots of people are hitting this, we can always change the logic to update links for all moved files so long as any of the files match `markdown.updateLinksOnFileMove.include`. Today we only update links for the files matching `markdown.updateLinksOnFileMove.include`

**Comment by mjbvz** [comment]
@andreamah When testing make sure you use the new setting name for the globs: `markdown.updateLinksOnFileMove.include`

If you still see issues, please share the exact structure of your workspace and note where you are moving the files so I can try to reproduce this

**Comment by andreamah** [comment]
I repro'd with a smaller example:

```
eeee/
|-> perf.png
subfolder/
test.md
```

test.md has:
```
[test](./eeee/perf.png)
```

And I move both at once to the empty `subfolder/`.
![Recording 2022-10-27 at 10 53 25](https://user-images.githubusercontent.com/31675041/198363193-8bd002af-ed92-4bc7-9829-0a0f3e30e612.gif)

I expect the new link to be something like `./perf.png` since `perf.png` isn't inside of `eeee/` anymore

**Comment by andreamah** [comment]
Here's what's in the debug console:
<img width="426" alt="image" src="https://user-images.githubusercontent.com/31675041/198386078-6c4a77a2-4739-4a65-b41d-70f8bf434bf9.png">

**Comment by mjbvz** [comment]
Likely fixed by #164829 but possibly also needs https://github.com/microsoft/vscode-markdown-languageservice/pull/82 (which I don't plan on taking into October)
